config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'products/index'
```

Comment:
Please remove this route. It's a duplicate of line 3

Reply:
Done